presto-tests/src/main/java/io/prestosql/tests/AbstractTestQueries.java

```diff
@@ -4617,6 +4618,8 @@ public void testAccessControl()
         assertAccessAllowed("SELECT name AS my_alias FROM nation", privilege("my_alias", SELECT_COLUMN));
         assertAccessAllowed("SELECT my_alias from (SELECT name AS my_alias FROM nation)", privilege("my_alias", SELECT_COLUMN));
         assertAccessDenied("SELECT name AS my_alias FROM nation", "Cannot select from columns \\[name\\] in table .*.nation.*", privilege("name", SELECT_COLUMN));
+        assertAccessDenied("show create table orders",  "Cannot show columns of table .*.orders.*", privilege("tiny.orders", SELECT_TABLE));
```

Comment:
Travis was failed by double whitespaces and let's use uppercase as existing test.
Also, I would suggest to update commit message. We don't use progressive (Adding) nor past (Added) tense as this [guideline](https://chris.beams.io/posts/git-commit/).
```suggestion
        assertAccessDenied("SHOW CREATE TABLE orders", "Cannot show columns of table .*.orders.*", privilege("tiny.orders", SELECT_TABLE));
```

Reply:
fixed that